src/main/java/net/dv8tion/jda/api/events/message/GenericMessageEvent.java

```diff
@@ -156,6 +156,18 @@ public Guild getGuild()
         return ((GuildChannel) channel).getGuild();
     }
 
+    /**
+     * Returns the jump-to URL for the received message. Clicking this URL in the Discord client will cause the client to
+     * jump to the specified message.
+     *
+     * @return A String representing the jump-to URL for the message
+     */
+    @Nonnull
+    public String getJumpUrl()
+    {
+        return String.format(Message.JUMP_URL, isFromGuild() ? getGuild().getId() : "@me", getChannel().getId(), getMessageId());
```

Comment:
This should use `Helpers.format` (you can change it in Message#getJumpUrl too)

Reply:
Fixed in f12efdc.